integration/build/build_test.go

```diff
@@ -143,7 +143,7 @@ func buildContainerIdsFilter(buildOutput io.Reader) (filters.Args, error) {
 // GUID path (\\?\Volume{dae8d3ac-b9a1-11e9-88eb-e8554b2ba1db}\newdir\hello}),
 // which currently isn't supported by Golang.
 func TestBuildMultiStageCopy(t *testing.T) {
-	ctx := testutil.StartSpan(baseContext, t)
+	ctx := setupTest(t)
```

Comment:
For future self (and others); `setupTest(t)` also takes care of taking the `baseContext`; https://github.com/moby/moby/blob/6a810b4259547c4867d1cb3d2cc84acba5fd65f4/integration/build/main_test.go#L51-L56